ckanext/data_qld/plugin.py

```diff
@@ -191,7 +191,8 @@ def after_search(self, search_results, search_params):
         return search_results
 
     def after_delete(self, context, data_dict):
-        dataset_deletion_helpers.add_deletion_of_dataset_reason(context, data_dict)
```

Comment:
Do you want to start making 2.10.x api endpoints so we are ahead? 

https://github.com/ckan/ckan/issues/6787
https://github.com/ckan/ckan/wiki/CKAN-2.9-to-2.10-migration-tips#ipackagecontroller-hooks-renamed

Reply:
Eh, that can come in a later round.